bro/index.html

```diff
@@ -169,37 +116,41 @@
       <p>Op <a href="https://docs.geostandaarden.nl/bro">https://docs.geostandaarden.nl/bro</a> publiceert Geonovum de standaarden en technische documenten voor de Basisregistratie Ondergrond (BRO).</p>
       <p class="warning">Deze pagina is slechts een inhoudsopgave van documentatie die wij beheren. Ga naar de <a href="https://www.bro-productomgeving.nl/bpo/latest/">BRO Productomgeving</a> voor toelichting op de documentatie.</p>
       <p>Onderstaande documenten zijn op dit moment beschikbaar:</p>
-     
-	  
-
-<?php 
-
-foreach ($config as $domein => $registratieobjecten) { 	
-	
-	echo '<div class="pubDomain">';	
-	
-	echo '<h2><a href="bro">' . $domein . '</a></h2>';
-	
-	if (count($registratieobjecten) == 0) {
-		echo '<i>Geen publicaties</i>';
-	}
-	foreach ($registratieobjecten as $mnemonic=>$naam) {
-		echo '<h3><a href="./' . $mnemonic . '">'.$naam . ' ('.strtoupper($mnemonic) . ')</a></h3>';
-	}
-	echo '</div>';
-}
-	
-	/* echo '<ul class="docs">';
-	foreach ($registratieobjecten as $mnemonic=>$naam) {
-		echo ' <li>';
-		echo '  <span class="final">';
-		echo '   <a href="./' . strtolower($mnemonic) . '">'.$naam . ' ('.$mnemonic . ')</a>';
-		echo '  </span>';
-		echo ' </li>';
-	}
-	echo '</ul>'; 
-}*/
-?>			 
-   
+<div class="pubDomain">
+  <h2><a href="bro">Domein Bodem- en grondonderzoek</a></h2>
+  <h3><a href="./bhr-g">Booronderzoek (BHR-G)</a></h3>
+  <h3><a href="./BHR-GT">Booronderzoek: Geotechnische boormonsterbeschrijving en boormonsteranalyse (BHR-GT)</a></h3>
```

Comment:
absolute paden hebben vanuit security perspectief de voorkeur. Al maakt het in dit geval weinig uit. De . voor de slash voegt echter ook weinig toe.